SandB-iOS/SandB-iOS/Controllers/ArticlePageViewHolderController.m

```diff
@@ -129,71 +112,24 @@ - (void)didReceiveMemoryWarning
     // Dispose of any resources that can be recreated.
 }
 
-
-- (void)loadReadingOptions
-{
-    self.fontSize = [[NSUserDefaults standardUserDefaults] floatForKey:@"ReadingOptionsFontSize"];
-    self.fontFamily = [[NSUserDefaults standardUserDefaults] objectForKey:@"ReadingOptionsFontFamily"];
-    self.isLightTheme = [[NSUserDefaults standardUserDefaults] boolForKey:@"ReadingOptionsIsLightTheme"];
-}
-
 #pragma mark - Page View Controller methods
 
-- (ArticleViewController *) viewControllerAtIndex:(NSUInteger) index {
+- (ArticleViewController *)viewControllerAtIndex:(NSUInteger) index {
```

Comment:
And most the style guides have no space, either their or between the type and index